wallet/src/actors/app/handlers/create_vtt.rs

```diff
@@ -65,41 +67,51 @@ impl Handler<CreateVttRequest> for app::App {
                         transaction,
                         bytes,
                         metadata: VttMetadata {
-                            to: msg.address,
-                            value: msg.amount,
                             fee: msg.fee,
-                            time_lock: msg.time_lock,
+                            outputs: msg.outputs,
                         },
                     }
                 })
+                .map_err(|err, _, _| {
+                    log::error!("Failed to create a VTT: {}", err);
+
+                    err
+                })
         });
 
         Box::new(f)
     }
 }
 
-/// Validate `CreateVttRequest`.
+/// Validate output addresses and transform addresses to `ValueTransferOutputs`
 ///
 /// To be valid it must pass these checks:
-/// - destination address must be in the same network (test/main)
-/// - source account must have enough balance
-fn validate(testnet: bool, address: &str) -> Result<PublicKeyHash, app::ValidationErrors> {
-    let pkh = validate_address(testnet, address)?;
-
-    Ok(pkh)
-}
+/// - destination address must be in the same network (testnet/mainnet)
+fn validate_output_addresses(
+    testnet: bool,
+    outputs: &[VttOutputParams],
+) -> Result<Vec<types::ValueTransferOutput>, app::ValidationErrors> {
+    outputs.iter().try_fold(vec![], |mut acc, output| {
+        types::PublicKeyHash::from_bech32(
+            if testnet {
```

Comment:
We can set the environment outside the loop right? that way we dont need to do the testnet mainnet comparison for every output